.github/workflows/screenshots.yml

```diff
@@ -0,0 +1,178 @@
+    - uses: actions/setup-ruby@v1
+      with:
+        ruby-version: '2.6.x'
+    
+    - name: Install Bundler
+      run: gem install bundler
+    
+    - name: Restore Ruby Dependency Cache
+      id: restore-ruby-dependency-cache
+      uses: actions/cache@v1
+      with:
+        path: vendor/bundle
+        key: ${{ runner.os }}-gems-${{ hashFiles('**/Gemfile.lock') }}
+    
+    - name: Install Ruby Dependencies
+      run: |
+        bundle config path vendor/bundle
+        bundle install --jobs 4 --retry 3
+    
+    - name: Restore CocoaPods Dependency Cache
+      id: restore-cocoapods-dependency-cache
+      uses: actions/cache@v1
+      with:
+        path: Pods
+        key: ${{ runner.os }}-pods-${{ hashFiles('**/Podfile.lock') }}
+
+    - name: Install App Dependencies
+      run: bundle exec rake dependencies
+    
+    - name: Compile the App
+      run: bundle exec fastlane build_screenshots
+
+    - name: Archive App
+      uses: actions/upload-artifact@v1
+      with:
+        name: screenshot-app
+        path: fastlane/DerivedData/Build/Products/Debug-iphonesimulator/WooCommerce.app
+
+    - name: Archive Runner
+      uses: actions/upload-artifact@v1
+      with:
+        name: screenshot-runner
+        path: fastlane/DerivedData/Build/Products/Debug-iphonesimulator/WooCommerceScreenshots-Runner.app
+
+  capture:
+    name: Capture
+    needs: build
+    runs-on: macos-latest
+
+    strategy:
+      matrix:
+        language: [ar, de-DE, en-US, es-ES, fr-FR, he, id, it, ja, ko, nl-NL, pt-BR, ru, sv, tr, zh-Hans, zh-Hant]
+        mode: [dark, light]
+    steps:
+    - uses: actions/checkout@v1
+    - uses: actions/setup-ruby@v1
+      with:
+        ruby-version: '2.6.x'
+    
+    - name: Install Bundler
+      run: gem install bundler
+    
+    - name: Restore Ruby Dependency Cache
+      id: restore-ruby-dependency-cache
+      uses: actions/cache@v1
+      with:
+        path: vendor/bundle
+        key: ${{ runner.os }}-gems-${{ hashFiles('**/Gemfile.lock') }}
+    
+    - name: Install Ruby Dependencies
+      run: |
+        bundle config path vendor/bundle
+        bundle install --jobs 4 --retry 3
+
+    - name: Install Fastlane Dependencies
+      run: bundle exec fastlane run configure_apply
+    
+    - name: Download Screenshot App
+      uses: actions/download-artifact@v1
+      with:
+        name: screenshot-app
+        path: fastlane/DerivedData/Build/Products/Debug-iphonesimulator/WooCommerce.app
+
+    - name: Download Screenshot Runner
+      uses: actions/download-artifact@v1
+      with:
+        name: screenshot-runner
+        path: fastlane/DerivedData/Build/Products/Debug-iphonesimulator/WooCommerceScreenshots-Runner.app
+
+    - name: Start Mocks
+      run: rake mocks &
+
+    - name: Generate Screenshots
+      run: |
+        bundle exec fastlane take_screenshots languages:${{ matrix.language }} mode:${{ matrix.mode }}
+
+    - name: Move Test Result Bundle to Logs
+      if: always()
```

Comment:
Is there a particular use case for `if: always()`?

Reply:
Yeah! That syntax causes the step to run even if the preceding steps fail – it's good for getting logs off a CI machine that's failed a step!